### ISSUE TITLE
Individual poses of reference database images and view synthesis

### ISSUE THREAD

**Issue by Shubodh** [HajimeTaira/InLoc_dataset#8]
I am looking for individual poses of reference database images.

I went through the Inloc dataset, it looks like the pose for every reference database image has not been provided. Instead, we have a `.mat` 3D scan associated with every ref image, using which we can apply [P3P](https://github.com/mihaidusmanu/pycolmap#absolute-pose-estimation) between 2D-3D pair and obtain the desired pose as a closed form solution.

However, the pose obtained this way seems to have a *slight* offset. 

The way I verified this is by: First using the 3D scan and our obtained pose (through p3p), I synthesized an image at that pose. Now since we already have our original image, I can compare the synthesized image and original image. [Here are the examples](https://saishubodh.notion.site/GitHub-issue-Individual-poses-of-reference-database-images-and-view-synthesis-3355b99e408641ba9ebf95023b6e4d1b).

So even this slight offset could result in more than 25% of the original image not visible in synthsized image.

1. Is such offset considered within tolerable limit?
2. Could you provide actual ground truth poses for reference images so that this offset will be avoided?

**Comment by lahavlipson** [comment]
@Shubodh Did you end up finding the ground truth camera poses anywhere? It seems that all the Inloc data [is available](https://data.ciirc.cvut.cz/public/projects/2020VisualLocalization/) except for the poses of the database/reference images. If not, did you end up using the poses solved from P3P? Thanks!

**Comment by Shubodh** [comment]
Yeah it seems like poses are not available. How unfortunate, would've been a complete dataset otherwise!

Yes, i ended up using poses from p3p, but as I shared in "verified" part of the original message, they don't seem extremely accurate.

**Comment by haoningc** [comment]
At https://cvpr17.wijmans.xyz/data/ have InLoc transformation matrix of the data, but only a transformation matrix corresponding to each scenario, rather than each figure corresponds to a transition matrix. I don't understand this. Could you help take a look and discuss it together?Thanks！ @Shubodh @lahavlipson
